api/hasura/actions/_handlers/deleteCircle.ts

```diff
@@ -0,0 +1,64 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import { authCircleAdminMiddleware } from '../../../../api-lib/circleAdmin';
+import { adminClient } from '../../../../api-lib/gql/adminClient';
+import { errorResponseWithStatusCode } from '../../../../api-lib/HttpError';
+import {
+  deleteCircleInput,
+  composeHasuraActionRequestBody,
+} from '../../../../src/lib/zod';
+
+async function handler(req: VercelRequest, res: VercelResponse) {
+  const {
+    input: { payload },
+  } = composeHasuraActionRequestBody(deleteCircleInput).parse(req.body);
+
+  const { circle_id } = payload;
+
+  const {
+    circles: [existingCircle],
+  } = await adminClient.query(
+    {
+      circles: [
+        {
+          limit: 1,
+          where: {
+            id: { _eq: circle_id },
+            // ignore soft_deleted circles
+            deleted_at: { _is_null: true },
+          },
+        },
+        { id: true },
+      ],
+    },
+    {
```

Comment:
please avoid splitting things like this onto multiple lines when it's not necessary

Reply:
I did the other changes but I don't understand this. you mean putting the braces and brackets in the same line? that changes automatically by lint after saving

Reply:
The linter won't put it on one line if you have it on multiple lines, only the reverse. In this case, this should all fit on one line:
```
{ operationName: 'deleteCircle_getExistingCircle' }
```